.yamllint.yml

```diff
@@ -26,3 +26,5 @@ rules:
   braces:
     min-spaces-inside: 1
     max-spaces-inside: 1
+  trailing-spaces: disable
```

Comment:
I don't have super strong opinions on this, but these do add consistency IMO. 
It should also be mentioned that you can configure most editors to correct these on save.

Reply:
I totally forgot that IDEs support this. I'm adding this rule back, thanks for the reminder!